### DISCUSSION TITLE
Интеграция РТЖ

### DISCUSSION THREAD

**Discussion by ValeryVerkhoturov** [0niel/university-app#201] [Идеи]
**Цель**
Вовлечение студентов в проблемы комплексного развития радиотехнических, телекоммуникационных и информационных систем, электроники и информатики путем ознакомления с Российским Технологическим Журналом.

**Решение**
* функция просмотра журналов из архива
* функция навигации по разделам журнала и просмотра отдельной статьи

**Дополнительно**
Архив https://www.rtj-mirea.ru/jour/issue/archive
Разделы https://www.rtj-mirea.ru/jour/about/editorialPolicies#sectionPolicies
Авторские права https://www.rtj-mirea.ru/jour/about/submissions#copyrightNotice

**Comment by 0niel** [comment]
Мне кажется, что этим никто не будет пользоваться. Целевая аудитория слишком узкая, и слишком уж сложная фича для такого маленького профита.

**Reply by d3adwolf** [reply]
Сыглы. :)
